CheckAllDependenciesIT.testNoMandatoryDependencyDeclared failure

*master* (commit a79e2bfe2dad0598536995e984adbe2244102059)
*maintenance* (commit 3be1639dd4b5e497950aede41a41fbadecbafc34)

Failed on various JDKs 8 (maintenance): https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-OracleJDK8/com.hazelcast$hazelcast-all/642/testReport/junit
/com.hazelcast.it/CheckAllDependenciesIT/testNoMandatoryDependencyDeclared/
Failed on various JDKs 8 (master): https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK8/com.hazelcast$hazelcast-all/1538/testReport/junit/com.hazelcast.it/CheckAllDependenciesIT/testNoMandatoryDependencyDeclared/

Stacktrace:
```
java.lang.AssertionError: Import com.eclipsesource.json is not declared as optional
```

--- a/hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/discovery/HazelcastCloudDiscovery.java
+++ b/hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/discovery/HazelcastCloudDiscovery.java
@@ -16,8 +16,8 @@
 
 package com.hazelcast.client.spi.impl.discovery;
 
-import com.eclipsesource.json.Json;
-import com.eclipsesource.json.JsonValue;
+import com.hazelcast.com.eclipsesource.json.Json;
+import com.hazelcast.com.eclipsesource.json.JsonValue;
 import com.hazelcast.client.util.AddressHelper;
 import com.hazelcast.nio.Address;
 import com.hazelcast.util.AddressUtil;

